code/matt/html_css_flask/lab02/static/styles.css

```diff
@@ -0,0 +1,170 @@
+@import url(https://fonts.googleapis.com/css?family=Raleway);
+
+body {
+  margin: 0;
+  background-color: bisque;
+  display: flex;
+  flex-direction: column;
+}
+
+.container {
```

Comment:
If you aren't using a css tag you should remove it or comment it out to avoid conflicts.